libs/wingsdk/src/target-tf-azure/bucket.inflight.ts

```diff
@@ -34,6 +34,19 @@ export class BucketClient implements IBucketClient {
     );
   }
 
+  /**
+   * Check if an object exists in the bucket
+   *
+   * @param key Key of the object
+   */
+  public async exists(key: string): Promise<boolean> {
+    key = "none";
+    throw new Error(
+      "exists method isn't implemented yet on the current target."
+    );
+    return false;
```

Comment:
I think this should fix the current build error:
```suggestion
    key;
    throw new Error(
      "exists method isn't implemented yet on the current target."
    );
```